journals/apps/journals/templates/wagtailadmin/404.html

```diff
@@ -0,0 +1,5 @@
+{% extends "wagtailadmin/404.html" %}
+
+{% block branding_logo %}
+    <img class="wagtail-logo" itemprop="logo" typeof="foaf:Image" src="//edx.org/sites/default/files/theme/edx-logo-header.png" alt="edX Journal Editor">
```

Comment:
why is the src in a strange format? 
src="//edx.org/sites/default/files/theme/edx-logo-header.png"

Reply:
omitted url scheme to use url scheme of journals.